warehouse/db.py

```diff
@@ -90,6 +90,11 @@ class Model(ModelBase):
         primary_key=True,
         server_default=sqlalchemy.text("gen_random_uuid()"),
     )
+    created = Column(
+        DateTime(timezone=False),
```

Comment:
Curious, not material to the refactor, why do we not store the timezone as part of the timestamp?

Reply:
🤷 This predates me: https://github.com/pypi/warehouse/blame/ead69a5793eb98452f7da7dc90bceda29b44c7f3/warehouse/packaging/models.py#L114